sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -318,6 +359,17 @@ export interface SendOptions {
     partitionKey?: string | null;
 }
 
+// @public
+export interface Subscription {
+    consumerGroup(): string;
```

Comment:
Why'd you choose to expose consumerGroupName and isReceiverOpen on this interface?

I can see `isReceiving` or `isRunning` being useful, but not as sure about the other two.